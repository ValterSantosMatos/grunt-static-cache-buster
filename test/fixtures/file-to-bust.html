<!-- Dummy file -->
<script type="text/javascript" src="/file-to-hash.js"></script>