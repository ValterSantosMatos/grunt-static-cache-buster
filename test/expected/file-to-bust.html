<!-- Dummy file -->
<script type="text/javascript" src="/file-to-hash-79d310bfe9ba9930c28043700467788e.js"></script>